data/releases.yml

```diff
@@ -214,15 +214,15 @@
   icon: S
   description: Apache SkyWalking Client-side JavaScript exception and tracing library.
   source:
-    - version: v0.2.0
-      date: Nov. 17th, 2020
+    - version: v0.3.0
+      date: Jan. 14th, 2021
       downloadLink:
         - name: src
-          link: https://www.apache.org/dyn/closer.cgi/skywalking/client-js/0.2.0/skywalking-client-js-0.2.0-src.tgz
+          link: https://www.apache.org/dyn/closer.cgi/skywalking/client-js/0.3.0/skywalking-client-js-0.3.0-src.tgz
         - name: asc
-          link: https://downloads.apache.org/skywalking/client-js/0.2.0/skywalking-client-js-0.2.0-src.tgz.asc
+          link: https://downloads.apache.org/skywalking/client-js/0.3.0/skywalking-client-js-0.3.0-src.tgz.asc
```

Comment:
I think asc and sha are not available on mirrors. Take a look at the main repo release.

Reply:
![image](https://user-images.githubusercontent.com/26432832/104549729-5fe27500-566e-11eb-9b9b-bbea892a7ad2.png)
It looks the same